src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  width: 7 * ($day-size + $gap) - $gap + 2 * $calendar-padding;
```

Comment:
It's recommended to use 'margin' instead of 'gap' when calculating the width to support older browsers that may not recognize the 'gap' property in flexbox. Consider using 'margin' in the calculation and in the '&__day' class.